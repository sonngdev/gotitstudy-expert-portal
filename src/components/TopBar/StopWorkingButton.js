import React from 'react';
import { ButtonWithModal } from '../shared';

export function StopWorkingButton() {
  return (
    <ButtonWithModal
      buttonProps={{
        variant: 'white_outline',
        size: 'small',
        width: 'full',
        className: 'stop-working',
      }}
      buttonAction="Stop working"
      modalBody="Are you sure you want to stop working?"
      onConfirm={() => window.location.reload()}
    />
  );
}
