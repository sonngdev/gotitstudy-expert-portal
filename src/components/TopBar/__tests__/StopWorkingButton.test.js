import React from 'react';
import { shallow } from 'enzyme';
import { StopWorkingButton } from '../StopWorkingButton';

describe('<StopWorkingButton />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<StopWorkingButton />);
  });

  it('renders', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
