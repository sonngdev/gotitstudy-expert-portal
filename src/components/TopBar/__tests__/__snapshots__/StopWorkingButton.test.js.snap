// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StopWorkingButton /> renders 1`] = `"<ButtonWithModal buttonProps={{...}} buttonAction=\\"Stop working\\" modalBody=\\"Are you sure you want to stop working?\\" onConfirm={[Function: onConfirm]} modalSize=\\"small\\" />"`;
